packages/@webex/plugin-authorization-browser-first-party/src/authorization.js

```diff
@@ -240,6 +242,128 @@ const Authorization = WebexPlugin.extend({
       });
   },
 
+  /**
+   * Get an OAuth Login URL for QRCode. Generate QR code based on the returned URL.
+   * @instance
+   * @memberof Credentials
+   * @returns {Object}
+   */
+  getQRCodeLoginDetails() {
+    return this.webex
+      .request({
+        method: 'POST',
+        service: 'oauth-helper',
+        resource: '/actions/device/authorize',
+        form: {
+          client_id: this.config.client_id,
+          scope: this.config.scope,
+        },
+        auth: {
+          user: this.config.client_id,
+          pass: this.config.client_secret,
+          sendImmediately: true,
+        },
```

Comment:
Since we just requesting the QR code details, I do not think we need `auth` parameter.

Reply:
I found that if auth is not added, the error "Current state cannot produce an access token" will be reported when making a request. I tried to remove the auth field in other existing methods and found that this error will also be reported. So I added the auth parameter, although this is indeed a bit strange and probably not the right solution
<img width="763" alt="image" src="https://github.com/user-attachments/assets/2a4f2940-06ab-450e-9115-7268887185f6">
<img width="732" alt="image" src="https://github.com/user-attachments/assets/2be1caae-3986-4089-9e92-1077194edb5d">

Reply:
strange, but OK, we can keep the `auth` parameter